### ISSUE TITLE
[Comprehension] User doesn't have enough information to complete a task.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#80109]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Benefits Non-Disability
**Product:** Burial Form 530
**Feature:** 21P-530 Parity Updates

## Findings details
**VA.gov Experience Standard - issue:** User doesn't have enough information to complete a task.
**VA.gov Experience Standard - category:** Comprehension
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [v2 /military-history/separation-documents](https://staging.va.gov/burials-and-memorials-v2/application/530/military-history/separation-documents), the helper text starting with "You can choose one of these options" and ending with "you can request them and finish this form later" may be missed by most screen reader users.

When moving through a form flow like this, screen reader users will typically either be using their software in forms mode or otherwise just be tabbing between interactive elements --- not having their screen reader announce the whole page. The implicit expectation is that any text that's necessary for completing an input will be programmatically associated with that input. But that means helper text that's technically accessible may be missed by screen reader users if it's not associated with any inputs.

If the helper text isn't critical to completing the task, then that's not a huge issue. But if it's important information that the user should really know when answering the questions on the page, we need to ensure that it's properly associated and announced.

## Recommended action

There are a couple of options for resolving this --- totally up to you for what feels right.

**Option 1:** Make all of the helper text part of the `legend` element and then style it so it doesn't look like the `legend` visually, eg.
```
<legend>
   <span class="some-class">DD214 or other separation documents</span>
   <span class="some-other-class">You can choose one of these options:</span>
   ...
</legend>
```

When a screen reader user tabs into the first interactive element in the `fieldset`, everything in the legend will be announced --- so nothing will be missed.

**Option 2:** As I read the helper text on this page, I think the only part that's must-read important is probably this part:
> Uploading a copy can help us process your application faster and you’ll skip certain questions about the deceased Veteran’s military service history.

That helps the user understand why the question matters and the potential consequences of their answer.

With that in mind, you could slightly regroup the content to move that portion (or something similar) to the `hint` prop for the `va-radio` component. The `hint` text is associated with the radio buttons via `aria-describedby` and will be announced by screen readers when tabbing to those radio buttons. All of the other helper text on this page will still likely be missed, but if it's not critical to deciding how to answer the question then I think that's fine.

## References

- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by sortizsh** [comment]
@DLarson-Oddball  can you pick this up please

**Comment by DLarson-Oddball** [comment]
@steele-lm Code for this issue has been merged. Thanks!
